distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1281,7 +1260,7 @@ type Table
            not possible to create a table without any columns.
          - If a given aggregate is not supported by the backend,
            `Unsupported_Database_Operation` is reported.
-         - If a column index is out of range, a `Column_Indexes_Out_Of_Range` is
```

Comment:
Here, an exception - the `Text` case has some specific explanation and behaves slightly differently here. So I keep the special case for 'index' because it _differs_ from Text here. I just update the comment to tell the current state.